src/puppetlabs/puppetdb/schema.clj

```diff
@@ -57,6 +58,9 @@
 (def Function
   (s/pred fn? 'fn?))
 
+(def JSONable
```

Comment:
A quick docstring would be nice here. Looks like Function wasn't doc'd either, wanna add that while you're in there?